CHANGELOG.adoc

```diff
@@ -1,5 +1,11 @@
 = Changelog
 
+== v2023.6.0 (unreleased)
+
+=== Dependency update
+
+- https://github.com/eclipse-sirius/sirius-components/issues/1936[#1936] [releng] Switch to cypress 12.10.0
```

Comment:
12.11.0 now

Reply:
Changelog corrected with the new version